internal/app/machined/internal/sequencer/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -238,21 +220,14 @@ func (d *Sequencer) Upgrade(req *machineapi.UpgradeRequest) error {
 			rootfs.NewUnmountPodMountsTask(),
 		),
 		phase.NewPhase(
-			"unmount system disk",
+			"unmount system disks",
+			rootfs.NewUnmountSystemDisksTask(constants.BootPartitionLabel),
 			rootfs.NewUnmountSystemDisksTask(constants.EphemeralPartitionLabel),
 		),
 		phase.NewPhase(
 			"unmount system disk bind mounts",
 			rootfs.NewUnmountSystemDiskBindMountsTask(devname),
 		),
-		phase.NewPhase(
-			"verify system disk not in use",
-			disk.NewVerifyDiskAvailabilityTask(devname),
```

Comment:
Update this to run this test against the ephemeral partition (e.g. `/dev/sda2` _NOT_ `/dev/sda as we do today)